### PR TITLE
Add support for fontFamily on iOS

### PR DESCRIPTION
**Pull request by jamesblasco**

### CODE REVIEW

File: packages/stripe_ios/ios/Classes/CardFieldView.swift [hunk 1]
```diff
@@ -216,13 +216,39 @@ class CardFieldView: NSObject, FlutterPlatformView, STPPaymentCardTextFieldDeleg
...
+                cardField.font = font ?? UIFont.systemFont(ofSize: fontSize)
+            } else {
+                cardField.font = UIFont.systemFont(ofSize: fontSize)
+            }
+        }
+    }
+    
+    func registerFont(_ fontFamily: String) {
+        let controller = UIApplication.shared.delegate?.window??.rootViewController as? FlutterViewController;
+        let bundle = Bundle.main
```
Comment (remonh87) [inline packages/stripe_ios/ios/Classes/CardFieldView.swift:238]: what if the font is located in a separate package and not the main bundle? I do not think this solution facilitates for multi package projects. Either we should fix this or we should name this as a limitation

**Review by remonh87 — CHANGES_REQUESTED** [review]
Have 2 questions / concerns thatt I would like to see answered for the rest LGTM

File: packages/stripe_ios/ios/Classes/CardFieldView.swift [hunk 2]
```diff
@@ -216,13 +216,39 @@ class CardFieldView: NSObject, FlutterPlatformView, STPPaymentCardTextFieldDeleg
...
             }
-            if let fontSize = cardStyle["fontSize"]  as? Int {
-                cardField.font = UIFont.systemFont(ofSize: CGFloat(fontSize))
-            }
+           
             if let placeholderColor = cardStyle["placeholderColor"]  as? String {
                 cardField.placeholderColor = UIColor(hexString: placeholderColor)
             }
             
+            let fontSize = CGFloat(cardStyle["fontSize"]  as? Int  ?? 18);
```
[the remark this answers is not in the document]
Comment (jamesblasco) [reply packages/stripe_ios/ios/Classes/CardFieldView.swift:224]: Imply it. 18 is the one given by default by Stripe

File: packages/stripe_ios/ios/Classes/CardFieldView.swift (hunk 1 above)
Comment (jamesblasco) [reply packages/stripe_ios/ios/Classes/CardFieldView.swift:238]: I would prefer to push it like this, or without the registerFont. And then implement it also for packages and also on Android.

**Review by remonh87 — APPROVED** [review]

File: packages/stripe_ios/ios/Classes/CardFieldView.swift (hunk 1 above)
Comment (remonh87) [reply packages/stripe_ios/ios/Classes/CardFieldView.swift:238]: let's ship it then :)

**Pull request merged**